M2/Macaulay2/packages/JSON.m2

```diff
@@ -275,6 +290,12 @@ doc ///
       option.
     Example
       toJSON(hashTable{"foo" => 1, "bar" => 2, "baz" => 3}, Sort => true)
+    Text
+      Developers interested in writing a @CODE "toJSON"@ method for a new class
+      @CODE "X"@ should write a @CODE "toJSON(JSONEncoder, X)"@ method.
```

Comment:
I'm still unclear on why this is necessary. Can the content of JSONEncoder be relegated to options instead?

Reply:
Possibly -- I'll play around with it

Reply:
Figured it out!  This is much nicer -- thanks for the suggestion!